### PR TITLE
 fix: function name in pagination document is wrong

### PR DESCRIPTION
**Pull request by Plane-walker**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by Youggls** [comment]
LGTM!

File: components/pagination/index.zh-CN.md [hunk 1]
```diff
@@ -23,5 +23,5 @@
 ### 事件
 | 事件名称 | 说明 | 回调参数 |
 | --- | --- | --- |
-| change | 页码改变的回调，参数是改变后的页码及每页条数 | Function(page, pageSize) | noop |
-| showSizeChange | pageSize 变化的回调 | Function(current, size) | noop |
+| onChange | 页码改变的回调，参数是改变后的页码及每页条数 | Function(page, pageSize) | noop |
```
Comment (tangjinzhou) [inline components/pagination/index.zh-CN.md]: ? 两处都不需要改动啊

Comment (Plane-walker) [reply components/pagination/index.zh-CN.md]: 函数名写错了，原来的没用，得再之前加上on

Comment (tangjinzhou) [reply components/pagination/index.zh-CN.md]: 不会有 on，如果你用的 JSX，需要加 on，那是 JSX 插件的规范，并不是 vue 和 组件库的规范

Comment (Plane-walker) [reply components/pagination/index.zh-CN.md]: data () { 
      pagination: {
        pageSize: 10,
        total: 0,
        onChange: current => this.changePage(current)
      }
    }
这是vue里的写法，但是改为change就没有用

Comment (tangjinzhou) [reply components/pagination/index.zh-CN.md]: 你这是在 table 中使用？，table 中的 pagination 配置 要遵循 JSX 规范

Comment (Plane-walker) [reply components/pagination/index.zh-CN.md]: 原来如此

**Pull request closed without merging**